### PR TITLE
Bugfix/force download local invalid files

### PR DESCRIPTION
**Pull request by mgallien**

### CODE REVIEW

File: src/libsync/filesystem.cpp [hunk 1]
```diff
@@ -98,7 +98,7 @@ bool FileSystem::verifyFileUnchanged(const QString &fileName,
 {
     const qint64 actualSize = getSize(fileName);
     const time_t actualMtime = getModTime(fileName);
-    if (actualSize != previousSize || (actualMtime != previousMtime && previousMtime > 0 && actualMtime > 0)) {
```
Comment (allexzander) [inline src/libsync/filesystem.cpp:101]: @mgallien Why do we have `actualSize != previousSize` inside braces &&-ded with `actualMtime > 0`? Doesn't this change the behavior?

Comment (mgallien) [reply src/libsync/filesystem.cpp:101]: the fact is that we do not want to consider a local file being changed during sync if its modified time is not valid

Comment (mgallien) [reply src/libsync/filesystem.cpp:101]: and yes I should have added automated tests and will add them once I have finished another fix

**Review by allexzander — APPROVED** [review]

**Pull request merged**